### ISSUE TITLE
Composed characters and autocomplete

### ISSUE THREAD

**Issue by trondtynnol** [giellatekno/neahttadigisanit#26]
Composed Unicode characters consisting of many code points are treated as separate characters by the autocomplete. This means that typing `мо` into the search bar in sanj first shows hits starting in `мо̄`, as the macron comes first in sorting, before any other characters. It also throws off the alignment of the macron, as the о is shown in bold types, but the macron is not.

This may be especially confusing to users as two of these long vowels, ӣ and ӯ, are represented as a single character, and thus show different behavior than the rest of the long vowels.

Desirable functionality would be to treat these compound characters like single characters, so that typing `мо` only shows matches with short о.

![image](https://github.com/giellatekno/neahttadigisanit/assets/55655546/9bc3e045-fa32-4d61-a9c0-55a260c33d4e)

**Comment by Phaqui** [comment]
Fixed as of 2591ace910060bf0b10044db8b05d360e5fe967e

**Comment by Phaqui** [comment]
When looking at the code, https://github.com/giellatekno/neahttadigisanit/blob/main/neahtta/neahtta/nds_lexicon/lexicon.py#L472
be aware that the comment strings may look strange on github, and maybe in the editor.

```
# makes the "о" an "о̄", which are two different characters)
```
That is what the line should like, i.e. a "short o", and a "long o" (o without macron, and o with macron).
